keywords/couchbaseserver.py

```diff
@@ -1004,6 +1005,19 @@ def load_sample_bucket(self, sample_bucket):
         log_info("Enabling sample bucket {}".format(sample_bucket))
         self.remote_executor.must_execute('sudo /opt/couchbase/bin/cbdocloader -c localhost:8091 -u Administrator -p password -b {} -m 200 -d /opt/couchbase/samples/{}.zip'.format(sample_bucket, sample_bucket))
 
+    def get_bucket_connection(self, cbs_url, bucket_name, ssl_enabled, cluster):
+        cbs_ip = host_for_url(cbs_url)
+        if ssl_enabled and cluster.ipv6:
+            connection_url = "couchbases://{}/{}?ssl=no_verify&ipv6=allow".format(cbs_ip, bucket_name)
+        elif ssl_enabled and not cluster.ipv6:
+            connection_url = "couchbases://{}/{}?ssl=no_verify".format(cbs_ip, bucket_name)
+        elif not ssl_enabled and cluster.ipv6:
+            connection_url = "couchbase://{}/{}?ipv6=allow".format(cbs_ip, bucket_name)
+        else:
+            connection_url = 'couchbase://{}/{}'.format(cbs_ip, bucket_name)
+        sdk_client = Bucket(connection_url, password='password')
```

Comment:
this may need to be updated with sdk3 syntax once the sdk2>sdk3 PR merged to master

Reply:
reference this change in the PR: https://github.com/couchbaselabs/mobile-testkit/pull/1767/files#diff-00df9fc1cf3ee67d63285b3b2dd10825eee6ca5fc68255313f60675338b9fa3dR968